### PR TITLE
FIX: fixes for create a new key flow

### PR DESCRIPTION
**Pull request by rastajpa**

### CODE REVIEW

File: src/pages/settings/settings.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h4>Learn where you can spend your crypto today.</h4>
         </ion-note>
       </button>
       <button ion-item class="create-button" *ngIf="walletsGroups && walletsGroups[0]" (click)="goToAddView()" translate>
-        Create or Import a Key
```
Comment (cmgustavo) [inline src/pages/settings/settings.html]: I'm not sure on removing "Import"

Comment (rastajpa) [reply src/pages/settings/settings.html]: in the ticket was specified to delete the import option of `configuration -> create or import a key`... if it  won't possible to import, so the label must change

Comment (rastajpa) [reply src/pages/settings/settings.html]: Fixed! Thanks for reviewing it

**Comment by couby1master** [comment]
https://t.me/couby1master

**Comment by couby1master** [comment]
https://t.me/couby1master

**Pull request merged**